.github/workflows/ci.yaml

```diff
@@ -0,0 +1,30 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - 'master'
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      -
+        name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      -
```

Comment:
We can fetch the branch name and set the tag while pushing.
- [get branch name](https://github.com/renproject/devops/blob/master/.github/workflows/update-docker-image.yml#L23-L26)
- [get tag](https://github.com/renproject/devops/blob/master/.github/workflows/update-docker-image.yml#L28-L36)
- [set tag before pushing](https://github.com/renproject/devops/blob/master/.github/workflows/update-docker-image.yml#L58)